cpp/oneapi/dal/algo/svm/backend/utils.hpp

```diff
@@ -19,10 +19,12 @@
 #include "oneapi/dal/table/row_accessor.hpp"
 #include "oneapi/dal/detail/error_messages.hpp"
 #include "oneapi/dal/backend/transfer.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
 
 namespace oneapi::dal::svm::backend {
 
 namespace daal_svm = daal::algorithms::svm;
+namespace interop = dal::backend::interop;
```

Comment:
please do not use namespace aliases in headers - at least you can do it in a function or class scope